src/language/fi.json

```diff
@@ -905,16 +907,16 @@
       },
       "error": {
         "abort-failed": "Keskeyttäminen epäonnistui",
-        "signing-failed": "Allekirjoittaminen epäonnistui tuntemattomaan virheeseen. Voit aloittaa allekirjoituksen uudestaan tai keskeyttää allekirjoittamisen.",
+        "signing-failed": "Allekirjoittaminen keskeytyi tuntemattomasta syystä. Voit aloittaa allekirjoituksen uudestaan tai keskeyttää allekirjoittamisen.",
         "name-does-not-match": "Allekirjoittajan nimi henkilövarmenteessa ei vastaa käyttäjän nimeä järjestelmässä."
       },
       "status": {
         "not-started": "Voit nyt allekirjoittaa {language-genitive} energiatodistuksen.",
-        "already-started": "Voit aloittaa energiatodistuksen allekirjoittamisen uudestaan tai keskeyttää allekirjoitus ja palata luonnostilaan.",
+        "already-started": "Voit aloittaa energiatodistuksen allekirjoittamisen uudestaan tai keskeyttää allekirjoittamisen ja palata luonnostilaan.",
         "start": "Käynnistetään {language} allekirjoitusprosessi.",
-        "digest": "Muodostetaan {language} energiatodistus allekirjoitettavaksi.",
+        "digest": "Muodostetaan {language} energiatodistusta.",
```

Comment:
Jos ennen oli "Muodostetaan suomenkielinen energiatodistus allekirjoitettavaksi", niin nyt tulee ei toimiva lause "Muodostetaan suomenkielinen energiatodistusta."

Reply:
joo tämä ei voi olla näin

Reply:
lupasin viimeksi genetiivin, mutta en joka h...tin sijamuotoja

Reply:
eli nyt pitää pärjätä perusmuodolla ja genetiivillä!!!

Reply:
eli {language} tuottaa kielen adjektiivin perusmuodon esim. suomenkielinen. Genetiivin (esim. suomenkielisen) saa avainsanalla: {language-genetive}